src/Samples/Masa.Maui/wwwroot/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, viewport-fit=cover" />
+    <title>Masa.Maui</title>
+    <base href="/" />
+
+    <link href="masa-blazor.min.css" rel="stylesheet" />
+    <link href="css/masa-blazor-pro.css" />
```

Comment:
this css does not work. rel="stylesheet" should be added. 
like this:<link href="css/masa-blazor-pro.css" rel="stylesheet" />